Formula/bucklespring.rb

```diff
@@ -0,0 +1,23 @@
+class Bucklespring < Formula
+  desc "Nostalgia bucklespring keyboard sound"
+  homepage "https://github.com/zevv/bucklespring"
+  url "https://github.com/zevv/bucklespring/archive/v1.4.0.tar.gz"
+  sha256 "b38b2c0ec905573a243cb9db543f8eeb92e4d040025ef338d8bcfcfd5ccd5348"
+  license "GPL-2.0-only"
+
+  depends_on "alure" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    inreplace "Makefile", "-Wall -Werror", "-Wall"
```

Comment:
We don't really like new formulae that require patching.
This should be submitted to upstream.

Also, I'm a bit worried, is this project actively maintained?
The last release was 4 years ago,  nevertheless, it has some fresher commits.

Reply:
upstream requested at https://github.com/zevv/bucklespring/pull/97